### DISCUSSION TITLE
How to setup icon based on filename ending pattern?

### DISCUSSION THREAD

**Discussion by ymn64** [kyazdani42/nvim-web-devicons#203] [Q&A] [answered]
I want to setup a different Go icon for files that end with `_test.go`.
I tried this:
```lua
['_test.go'] = { --[[ ... ]] },
```
And: 
```lua
['_test.go$'] = { --[[ ... ]] },
```
Doesn't work.

**Comment by gegoune** [comment, marked as answer]
Please have a look if `override` works for your use case (https://github.com/nvim-tree/nvim-web-devicons/issues/36#issuecomment-850872669).
